app/imports/templates/components/nameStatus.js

```diff
@@ -144,36 +144,38 @@ Template['components_nameStatus'].helpers({
     },
     showExpiring() {
       var revealDeadline = Math.floor(new Date().getTime()/1000) + 48 * 60 * 60;
-      return Names.find({registrationDate: {$gt: revealDeadline}, name:{$gt: '', $regex: /^.{7,}$/}},{sort: {registrationDate: -1}}).count() > 100;
+      return Names.find({registrationDate: {$gt: revealDeadline, $lt: revealDeadline + 24 * 60 * 60}, name:{$gt: '', $regex: /^.{7,}$/}},{sort: {registrationDate: 1}, limit: 48}).count() > 1;
     }, 
     publicAuctionsAboutToExpire() {
       // subtracts 10 minutes from the reveal deadline, for good measure
       var revealDeadline = Math.floor(new Date().getTime()/1000) + 48 * 60 * 60 + 10 * 60;      
-      return Names.find({registrationDate: {$gt: revealDeadline}, name:{$gt: '', $regex: /^.{7,}$/}},{sort: {registrationDate: 1}, limit: 48});
+      return Names.find({registrationDate: {$gt: revealDeadline, $lt: revealDeadline + 24 * 60 * 60}, name:{$gt: '', $regex: /^.{7,}$/}},{sort: {registrationDate: 1}, limit: 48});
     }, 
     knownNamesRegistered() {
-      return Names.find({registrationDate: {$lt: Math.floor(Date.now()/1000)}, mode: {$nin: ['open', 'forbidden', 'not-yet-available']}, name:{$gt: ''}},{sort: {registrationDate: -1}, limit: 99});
+      return Names.find({registrationDate: {$lt: Math.floor(Date.now()/1000)}, mode: {$nin: ['open', 'forbidden', 'not-yet-available']}, name:{$gt: ''}},{sort: {registrationDate: -1}, limit: 48});
     },
     namesRegistered() {
-      return Names.find({value: {$gt:0}, mode: {$nin: ['open', 'forbidden', 'not-yet-available']}}).count();
+      return Names.find({value: {$gt:0}, mode: {$nin: ['open', 'forbidden', 'not-yet-available']}}).count() > 1;
     }, 
     hasAuctions() {
       var revealDeadline = Math.floor(new Date().getTime()/1000) + 48 * 60 * 60;      
       return Names.find({registrationDate: {$gt: revealDeadline}, name:{$gt: ''}},{}).count() > 0;
     },
-    averageValue() {
-      var average = _.reduce(
-          Names.find({value: {$gt:0.01}}).fetch(), function(memo,num) { 
-            return memo + num.value; 
-          }, 0);
-      return Math.round(average*100/Names.find({value: {$gt:0.01}}).count())/100 || '--';
+    medianValue() {
+      var disputedNames = Names.find({value: {$gt:0.01}}, {sort: {value: 1}}).fetch();
+      if (!disputedNames) return '---';
+      return disputedNames[Math.floor(disputedNames.length/2)].value;
     }, 
     percentageDisputed() {
       return Math.round(100 - (100 * Names.find({value: {$gt:0.01}}).count() / Names.find({value: {$gt:0}}).count())) || 0;
     },
     canBeInvalidated(name) {
       return name.length < 7;
     },
+    recent(registrationDate) {
+      var diff = Math.floor(new Date().getTime()/1000) - registrationDate;      
+      return ((diff < 600 && diff > 0)|| (diff < 60-5*24*60*60)) ? 'recent' : '';
```

Comment:
Never mind, I see how the second condition could be true when the first isn't; how is that 'recent', though?